src/CollectionList.js

```diff
@@ -173,7 +173,21 @@ class CollectionList extends RootElem {
 
 		for (let item of this.collection) {
 			let component = this.componentFactory(item, idx);
-			let li = document.createElement(this.subTagName);
+
+			let li;
+
+			if (this.subTagName != null) {
+				li = document.createElement(this.subTagName);
+
+				this._rel.appendChild(li);
+
+				if (component) {
+					component.render(li);
+				}
+			} else {
+				li = component.render(this._rel);
```

Comment:
What about components that doesn't return an element, or components that renders multiple components into `this._rel`?

Perhaps good enough for now, but maybe we should leave a comment to ourselves.

Or even better, maybe we can use [`this._rel.childElementCount`](https://www.w3schools.com/jsref/prop_element_childelementcount.asp) before calling `render`, and then check afterward how many (if any) elements were added.
* If 1, we take it and put it in  `li`.
* Anything else, we throw an exception! I think we can demand that, if you don't use a subTagName, the child component MUST render just a single element; no more, no less.